graphql-server/src/utils/oncotree.ts

```diff
@@ -110,3 +112,50 @@ async function getOncotreeCodesFromNeo4j() {
     await session.close();
   }
 }
+
+/**
+ * Include cancer type fields in the search query indirectly by searching the Oncotree cache
+ * for matching cancerType or cancerTypeDetailed values. If a match is found, add the corresponding
+ * Oncotree code to the original search query.
+ */
+export function includeCancerTypeFieldsInSearch(
```

Comment:
This function includes cancer type fields in the search query indirectly by searching the Oncotree cache for matching cancerType or cancerTypeDetailed values. If a match is found, it adds the corresponding Oncotree code to the original search query, which is passed to the database query.